### PR TITLE
added logic for creating GO-KIT app and package

### LINKED ISSUE #2: Add GO-KIT project type
YoGo should be able to create new type of project
- GO-KIT project
- GO-KIT package

### PR DESCRIPTION
**Pull request by andriylesch**
close #2

### CODE REVIEW

File: generators/templates/pkg-gokit-endpoint/_endpoint_test.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	service.On("Get", testObjRequest).Return(testObjResponse, nil)
+
+	endpoint := MakeEndpoint(service)
+
+	//Act
+	wl, err := endpoint(nil, testObjRequest)
+
+	//Assert
+	assert.NoError(t, err)
+	assert.Nil(t, err)
```
Comment (peppelan) [inline generators/templates/pkg-gokit-endpoint/_endpoint_test.go]: line 20 and 21 are the same

File: generators/templates/pkg-gokit-endpoint/_http_transport.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	return <%- packagename%>Request{}, errors.New("Parameter 'id' is not found")
+}
+
+func encodeResponse(ctx context.Context, w http.ResponseWriter, response interface{}) error {
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	return json.NewEncoder(w).Encode(response)
+}
+
+func encodeError() kithttp.ErrorEncoder {
```
Comment (peppelan) [inline generators/templates/pkg-gokit-endpoint/_http_transport.go]: this fn would be better not be generated by default, as there are better ways to transform Go errors to HTTP responses

File: generators/templates/pkg-gokit-endpoint/_service.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"context"
+)
+
+type Service interface {
+	Get(ctx context.Context, request <%- packagename%>Request) (<%- packagename%>Response, error)
+}
+
+//Service struct
```
Comment (peppelan) [inline generators/templates/pkg-gokit-endpoint/_service.go]: // Service

**Review by peppelan — APPROVED** [review]
Suggestions are minor, looks good to me

File: generators/templates/pkg-gokit-endpoint/_endpoint_test.go (hunk 1 above)
Comment (andriylesch) [reply generators/templates/pkg-gokit-endpoint/_endpoint_test.go]: done

File: generators/templates/pkg-gokit-endpoint/_service.go (hunk 3 above)
Comment (andriylesch) [reply generators/templates/pkg-gokit-endpoint/_service.go]: done

File: generators/templates/pkg-gokit-endpoint/_http_transport.go (hunk 2 above)
Comment (andriylesch) [reply generators/templates/pkg-gokit-endpoint/_http_transport.go]: done

**Pull request merged**